### PR TITLE
create_gif

### LINKED ISSUE #64: Common: tool to record drone movement in gif files.
imageio looks good 
https://www.idiotinside.com/2017/06/06/create-gif-animation-with-python/

Env-NPImage class has a way to convert array to bytes, so everything is in memory.

### PR DESCRIPTION
**Pull request by Gorodecki**
tool to record drone movement in gif files

### CODE REVIEW

File: Stage1/common/create_gif.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        if file_name.endswith('.png'):
+            file_path = os.path.join(png_dir, file_name)
+            try:
+                images.append(imageio.imread(file_path))
+            except ValueError:
+                pass
+    imageio.mimsave(os.path.join(png_dir, 'movie.gif'), images, fps=fps)
+    print('Gif create.')
+
+create_gif('C:/Users\Anton\PycharmProjects\Project-bI\Media\png')
```
Comment (mjh-sakh) [inline Stage1/common/create_gif.py:17]: Don't include code test in file. As well it's absolute path, witch won't work for others.

File: Stage1/common/create_gif.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import os
+import imageio
+import natsort  # for actually sort file
+
+def create_gif(png_dir='../Media/png', fps=24):
+    images = []
+    for file_name in natsort.natsorted(os.listdir(png_dir)):
+        if file_name.endswith('.png'):
+            file_path = os.path.join(png_dir, file_name)
+            try:
```
Comment (mjh-sakh) [inline Stage1/common/create_gif.py:10]: Why do you think it may not work? Bad file?

File: Stage1/common/create_gif.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+                images.append(imageio.imread(file_path))
+            except ValueError:
+                pass
+    imageio.mimsave(os.path.join(png_dir, 'movie.gif'), images, fps=fps)
```
Comment (mjh-sakh) [inline Stage1/common/create_gif.py:14]: Consider adding some variability in the name so new result doesn't necessary override last one. 
Probably include it as a parameter for function.

**Review by mjh-sakh — APPROVED** [review]

**Comment by olegboev** [comment]
It is possible to create filenames with leading zeros, which allow sorting them using standard method. This removes the dependency on natsort library. Here is the code example:

```
# frame_number holds the frame number
cv2.imwrite(fr'Media\png\{frame_number:03d}.png', image)
```

**Comment by mjh-sakh** [comment]
As Anton is temporary out, I am merging it so others can incorporate changes.

**Pull request merged**